lib/membrane/rtp/ssrc_router.ex

```diff
@@ -286,13 +305,53 @@ defmodule Membrane.RTP.SSRCRouter do
     )
   end
 
-  defp emit_packet_arrival_event(payload, pad, ctx) do
+  defp register_inbound_frame_event(pad, ctx) do
+    Membrane.TelemetryMetrics.register(
+      @marker_received_telemetry_event,
+      ctx.pads[pad].options.telemetry_label
+    )
+  end
+
+  defp emit_rtcp_arrival_event(destination, ctx) do
+    Membrane.TelemetryMetrics.execute(
+      @rtcp_arrival_event,
+      %{},
+      %{},
+      ctx.pads[destination].options.telemetry_label
+    )
+  end
+
+  defp emit_rtcp_sent_event(destination, ctx) do
+    Membrane.TelemetryMetrics.execute(
+      @rtcp_sent_event,
+      %{},
+      %{},
+      ctx.pads[destination].options.telemetry_label
+    )
+  end
+
+  defp emit_packet_arrival_events(actions, ctx) do
+    for action <- actions do
+      with {:buffer, {pad, buffer}} <- action do
+        emit_packet_arrival_event(buffer, pad, ctx)
+      end
+    end
```

Comment:
1. `for` filters only matching elements
2. I'd return `:ok` since the result of for is not intended to be used outside
```suggestion
    for {:buffer, {pad, buffer}} <- actions do
      emit_packet_arrival_event(buffer, pad, ctx)
    end
    
    :ok
```